lib/maze/client/selenium/bs_browsers.yml

```diff
@@ -54,6 +60,12 @@ safari_10:
   os: "OS X"
   osVersion: "sierra"
 
+safari_11:
+  browserName: "safari"
+  browserVersion: "11.1"
+  os: "OS X"
+  osVersion: "sierra"
```

Comment:
Looking at the BrowserStack capabilities creator, this doesn't appear as a valid combination, is the `osVersion` meant to be `High Sierra` ?

Reply:
🤷‍♀️ no idea! I don't know they let you specify this if there's only one valid combination lol